decidim-admin/spec/commands/create_category_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+module Decidim
+  module Admin
+    describe CreateCategory, :db do
+      describe "call" do
+        let(:organization) { create(:organization) }
+        let(:participatory_process) { create :participatory_process, organization: organization }
+        let(:form_params) do
+          {
+            "category" => {
+              "name_en" => "Title",
+              "name_es" => "Title",
```

Comment:
Please translate this or maybe it's time to create a test helper similar to what @oriolgual did on the seeds (or maybe the same exact one?). There's lots of places where we're doing this and it feels wrong.